elevate-cpanel

```diff
@@ -2756,6 +2759,64 @@ sub _blocker_ea4_profile( $self ) {
     EOS
 }
 
+sub _blocker_script_updated ($self) {
+
+    my ( $should_block, $blocker_text ) = ( undef, undef );
+
+    if ( !$self->getopt('skip-elevate-version-check') ) {
```

Comment:
can't you just return here?

Reply:
This is the legacy of previous revisions of this logic.